microcosm_pubsub/producer.py

```diff
@@ -249,6 +256,8 @@ def configure_sns_topic_arns(graph):
     endpoint_url=None,
     mock_sns=True,
     skip=None,
+    # the size used to determine batching in the deferred batch producer
+    deferred_batch_size=typed(int, default_value=100),
```

Comment:
A bit awkward, but since the `DeferredBatchProducer` isn't a `graph`'d component, it seems harder to configure via this mechanism.